tests/integration_tests/test_automl.py

```diff
@@ -232,8 +228,7 @@ def test_train_with_config_remote(fs_protocol, bucket, test_data_tabular_large,
     }
 
     with remote_tmpdir(fs_protocol, bucket) as tmpdir:
-        with pytest.raises(ValueError) if not _ray200 else contextlib.nullcontext():
```

Comment:
@ShreyaR I removed this context manager because we are moving to support only Ray>=2.0. Wanted to call this out in case you think there are issues with this.